openwisp_radius/api/views.py

```diff
@@ -468,6 +469,8 @@ def post(self, request, *args, **kwargs):
         user = self.get_user(serializer, *args, **kwargs)
         token, created = UserToken.objects.get_or_create(user=user)
         radius_token = self.get_or_create_radius_token(user)
+        user.last_login = timezone.now()
+        user.save(update_fields=['last_login'])
```

Comment:
I tried commenting out these two lines and the test passes anyway, can you investigate why?
I thought this wasn't being done by the token serializer but I may have been wrong

Reply:
Amazing, lol, I'll check it out! :smile:

Reply:
The testcase had a problem; fixed it.

Reply:
from now on I expect you to do this on every test you write, that's how it should be done :blush: